package.json

```diff
@@ -177,6 +181,8 @@
     "test": "NODE_PATH=node_modules jest --config=jest.config.js --no-cache",
     "test:watch": "npm test -- --watch",
     "testing": "NODE_ENV=testing webpack-dev-server --config ./webpack/webpack.dev.js --progress --colors --content-base ./build --host 0.0.0.0 --port $npm_package_config_port",
+    "tsc": "NODE_ENV=development $(npm bin)/tsc",
+    "tsc:watch": "NODE_ENV=development $(npm bin)/tsc -w --noEmit",
```

Comment:
Why do you need the npm bin subshell?

Reply:
I find it more elegant than to directly access ./node_modules/bin. Would you do it that way?

Reply:
You don't need that at all, npm adds ./node_modules/.bin to your path so you can simply write tsc